### PR TITLE
Introduce PR and issue templates

### PR DESCRIPTION
**Pull request by MarcBruins**
This PR introduces issue and PR templates. If you are wondering what is looks like check out gitter channel.
I've used the templates from MvvmCross and i hope that @CheeseBaron is ok with it, if not i'll make my own.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/c-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+
+name: 🚀 Feature Request
+about: Want to see something new included in the Framework? Submit it!
```
Comment (MaxMommersteeg) [inline .github/ISSUE_TEMPLATE/c-feature-request.md]: **Framework** seems a bit off here.

File: .github/ISSUE_TEMPLATE/c-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## 🚀 Feature Requests
+
+<!--- Summary description of the feature --->
+
+### Contextualize the feature
+<!--- Where does it fit in the framework and why it's being included there. --->
```
Comment (MaxMommersteeg) [inline .github/ISSUE_TEMPLATE/c-feature-request.md]: Same as above. Maybe replace `framework` for `NuKeeper`?

Comment (MarcBruins) [reply .github/ISSUE_TEMPLATE/c-feature-request.md]: good spot

**Comment by Cheesebaron** [comment]
Just for transparency. I gave my consent to use those templates in a private chat 👍

**Pull request merged**